internal/mlflow_test.go

```diff
@@ -132,7 +132,7 @@ func deleteModel(t *testing.T, w *databricks.WorkspaceClient, ctx context.Contex
 
 func TestAccRegistryWebhooks(t *testing.T) {
 	ctx, w := workspaceTest(t)
-
+	w.Config.FixHostIfNeeded()
```

Comment:
i think it's better to add `CanonicalHostname()` method that will call `fixHostIfNeeded` internally, rather than flipping `fixHostIfNeeded` public. it's not intended for a public use.